idd/Energy+.idd.in

```diff
@@ -85322,7 +85330,122 @@ Table:TwoIndependentVariables,
   N720,N721,N722,N723,N724,N725,N726,N727,N728,N729,N730,N731,N732,N733,N734,N735,N736,N737,N738,N739, \note fields as indicated
   N740,N741,N742,N743,N744,N745,N746,N747,N748,N749,N750,N751,N752,N753,N754,N755,N756,N757,N758,N759, \note fields as indicated
   N760,N761,N762,N763,N764,N765,N766,N767,N768,N769,N770,N771,N772,N773,N774,N775,N776,N777,N778,N779, \note fields as indicated
-  N780,N781,N782,N783,N784,N785,N786,N787,N788,N789,N790,N791,N792,N793,N794,N795,N796,N797,N798,N799; \note fields as indicated
+  N780,N781,N782,N783,N784,N785,N786,N787,N788,N789,N790,N791,N792,N793,N794,N795,N796,N797,N798,N799, \note fields as indicated
```

Comment:
The new example file only goes to N565 and the object is extensible, so the only reason to extend this in the IDD would be to edit a longer table object in IDF Editor.  My guess is that most users will prefer the external file format, so I would drop the extra fields here and leave it at N799.